### ISSUE TITLE
StealJS.com audit

### ISSUE THREAD

**Issue by matthewp** [stealjs/stealjs#15]
@chasenlehara commented on [Tue Nov 15 2016](https://github.com/stealjs/steal/issues/923)

Go through the website and look for broken demos & links, missing images & videos, spelling & grammar issues, accessibility issues, poor performance, etc.
